go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/aquasecurity/trivy
 
 go 1.24
 
+replace github.com/aquasecurity/trivy-kubernetes v0.7.0 => github.com/afdesk/trivy-kubernetes v0.0.0-20250326124332-3807b652b635
```

Comment:
forgot to cleanup

Reply:
This PR still needs https://github.com/aquasecurity/trivy-kubernetes/pull/451 (where the actual functionality lies) to be merged first that's why it's still here.

Reply:
yeah I'm just saying we can't merge it with this right?